test/verify/check-machines

```diff
@@ -123,6 +123,10 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         self.addCleanup(self.browser.logout)
         self.allow_authorize_journal_messages()
 
+        wait(lambda: "4" in self.machine.execute("virsh net-list --all | wc -l"), delay=3)
+        wait(lambda: "3" in self.machine.execute("virsh pool-list --all | wc -l"), delay=3)
+        wait(lambda: "3" in self.machine.execute("virsh list --all | wc -l"), delay=3)
```

Comment:
These magic numbers are awkward, let's please not make assumptions about the table formatting. Please call these with `--name`, append `.strip()`, and compare for exact equality `== '0`'. We don't want this to succeed if there are still 13 domains or so.

Reply:
--name of course, good point